### DISCUSSION TITLE
is the configureGenericAuthRoles support custom `secret_id` configuration?

### DISCUSSION THREAD

**Discussion by muspelheim** [bank-vaults/bank-vaults#2426] [Q&A] [answered]
It seems that bank-vaults currently do not support the customization of a secret_id as outlined in the [official Vault documentation](https://developer.hashicorp.com/vault/api-docs/auth/approle#create-custom-approle-secret-id). If feasible, could you guide me on how to define it?

**Comment by akijakya** [comment, marked as answer]
Hi @muspelheim, thanks for using Bank-Vaults! The AppRole auth method is unfortunately not supported currently (you can check supported auth methods [here](https://github.com/bank-vaults/vault-sdk/blob/7090d894d587e7943a6f6a6809c260679602bac1/vault/client.go#L157-L186)), but if this feature would be useful for you, this would be a great way to help and contribute to the project! We are here to assist.
